instructions/scivis.tex

```diff
@@ -83,11 +83,12 @@ \section{The Tasks}
   \item Show that the maximum intensity projection compositing method works, show the implementation, and explain how this works
   \item Using the front-to-back compositing method, use your transfer function editor to interactively manipulate the transfer function to show the lab assisstant what is on the inside of the dataset.  Your editor must be capable of the following:
   \begin{todolist}
-    \item Have a single transfer function editor
     \item Change the transfer function without needing to reload the webpage
-    \item Have the ability to change the color and opacity mapping for volume samples
-    \item Be able to interact with the editor and change the transfer function settings with the mouse
-    \item Have the ability to simulate isosurface rendering by being able to create ``spiky'' transfer functions that have a high opacity only for a few values with different colors.
+    \item Have the ability to change the color and opacity mapping for volume samples, to any color
+    \item Be able to interact with the editor and change the transfer function settings using the mouse
+    \item Have the ability to simulate isosurface rendering by being able to create "spiky" transfer functions that have a high opacity only for a few values with different colors
```

Comment:
"spiky" is being highlighted on GitHub for some reason?

Reply:
Probably due to the quotations. I'll use the ` ``spiky'' ` version instead